modules/database/src/template/top/exampleApp/src/devXxxSoft.c

```diff
@@ -21,42 +21,30 @@
 #include "xxxRecord.h"
 #include "epicsExport.h"
 
-/*Create the dset for devXxxSoft */
-static long init_record();
-static long read_xxx();
-struct {
-    long        number;
-    DEVSUPFUN   report;
-    DEVSUPFUN   init;
-    DEVSUPFUN   init_record;
-    DEVSUPFUN   get_ioint_info;
-    DEVSUPFUN   read_xxx;
-}devXxxSoft={
-    5,
-    NULL,
-    NULL,
-    init_record,
-    NULL,
-    read_xxx,
-};
-epicsExportAddress(dset,devXxxSoft);
-
-
-static long init_record(pxxx)
-    struct xxxRecord    *pxxx;
+static long init_record(dbCommon *prec)
 {
```

Comment:
I wonder, 16 years after 39eaa0e335ae6d84db02e1e0d8e26d4a19758893, if this is the last of the K&R prototypes in Base?